### PR TITLE
LPS-121105 [Bug] DXP logo shown in Segments CE build

### PR DESCRIPTION
**Pull request by dgarciasarai**
**Description**

A DXP icon appears when creating/editing a Segment in the portal (no difference between CE and DXP). We remove that icon from the edition because you can only edit Liferay Segments through the Segments Editor, in other words, you cannot edit Analytics Cloud segments in the Segments Editor. See [LPS-121105](https://issues.liferay.com/browse/LPS-121105). 

Screenshot 2020-09-28 at 16 12 09

Also, we add some logic to differentiate CE or DXP, when the user has the portal connected with Analytics Cloud, in the Segments list (see Screenshots). If AC is not connected, the column `Source` does not appear.

**Screenshots**

* Not connected with AC (doesn't matter CE or DXP)

Screenshot 2020-09-28 at 12 15 59

* Connected with AC and Liferay DXP

Screenshot 2020-09-28 at 15 45 34

* Connected with AC and Liferay CE

Screenshot 2020-09-28 at 12 15 37

**Note**

- With `buildLang` gradle task, I generate also translations for `assign-site-roles` message key

### CODE REVIEW

**Comment by dgarciasarai** [comment]
Hey @cgoncas @darquesdev, can you take a look at this code? To differentiate between CE and DXP instance, what I used comes from these suggestions: https://liferay.slack.com/archives/C0FS5NMJN/p1598966611035300. I just review this with Julia and it's ok from her side.

Thanks! 😄

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
Hey @darquesdev and @dgarciasarai, we can't send this pull request right now, since a new key is added, and we can't translate it in time for 7.3 GA6

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -89,10 +91,20 @@ request.setAttribute("view.jsp-eventName", eventName);
...
 						<c:otherwise>
-							<liferay-ui:icon
-								message="source.dxp"
-								src='<%= PortalUtil.getPathContext(request) + "/assets/dxp-icon.svg" %>'
-							/>
+							<c:choose>
+								<c:when test='<%= releaseInfo.contains("CE") %>'>
+									<liferay-ui:icon
+										message="source.ce"
+										src='<%= PortalUtil.getPathContext(request) + "/assets/ce-icon.svg" %>'
```
Comment (cgoncas) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/view.jsp:98]: we should try never to use Util classes in JSPs.

Comment (darquesdev) [reply modules/apps/segments/segments-web/src/main/resources/META-INF/resources/view.jsp:98]: We could add a method in SegmentsDisplayContext to provide the proper icon (message and src)

**Comment by dgarciasarai** [comment]
Hey @cgoncas @darquesdev, I just fixed the frontend tests. We should remove the use of `PortalUtil` in the JSP. Backend needed! 😄  Thanks!

**Comment by 4lejandrito** [comment]
See https://github.com/liferay-tango/liferay-portal/pull/1181.

**Pull request closed without merging**